artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/MirrorTransaction.java

```diff
@@ -49,4 +51,13 @@ protected synchronized void afterCommit(List<TransactionOperation> operationsToC
       }
    }
 
+   @Override
+   public boolean isAllowPageTransaction() {
+      return ignorePageTransaction;
+   }
+
+   public MirrorTransaction setIgnorePageTransaction(boolean ignorePageTransaction) {
+      this.ignorePageTransaction = ignorePageTransaction;
+      return this;
+   }
```

Comment:
The naming of the methods, and the variable feels off. The variable defaults to false, so ignorePageTransaction=false, meaning the return of 'isAllowPageTransaction()' is false. So it 'isnt ignored', but it also 'isnt allowed'? Confusing. Is there a better name here for one of them to make clearer?

Reply:
ohhh.... I renamed the method and forgot to rename that one.. thanks